tests/e2e/specs/Pools.spec.js

```diff
@@ -15,6 +15,25 @@ import {
 
 import { nodeTypes } from '../support/constants';
 
+function boundaryEventIsCloseToTask() {
+  const positionErrorMargin = 30;
+
+  const taskSelector = '.main-paper ' +
+    '[data-type="processmaker.components.nodes.task.Shape"]';
+
+  const boundaryTimerEventSelector = taskSelector +
+    ' ~ [data-type="processmaker.components.nodes.boundaryEvent.Shape"]';
```

Comment:
This line, without the `~`, is exported in `tests/e2e/support/constants.js`:
https://github.com/ProcessMaker/modeler/blob/81ab6700ae8ae9baa5f137380d8f7f0996e13610/tests/e2e/support/constants.js#L42-L43

Maybe you can update that and use that export here as well?